FBC/FinancialInstruments/InstrumentPricing.rdf

```diff
@@ -0,0 +1,478 @@
+		<fibo-fnd-utl-av:synonym xml:lang="en">ask price</fibo-fnd-utl-av:synonym>
+		<fibo-fnd-utl-av:synonym xml:lang="en">asking price</fibo-fnd-utl-av:synonym>
+		<fibo-fnd-utl-av:synonym xml:lang="en">offering price</fibo-fnd-utl-av:synonym>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-fbc-fi-ip;OfficialClosingPrice">
+		<rdfs:subClassOf rdf:resource="&fibo-fbc-fi-ip;MarketPrice"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fbc-fi-ip;hasClosingPriceDeterminationMethod"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fbc-fi-ip;ClosingPriceDeterminationMethod"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label xml:lang="en">official closing price</rdfs:label>
+		<skos:definition xml:lang="en">price of the final trade of a security at the end of a trading day on a given exchange</skos:definition>
+		<fibo-fnd-utl-av:synonym xml:lang="en">end-of-day price</fibo-fnd-utl-av:synonym>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-fbc-fi-ip;OpeningPrice">
+		<rdfs:subClassOf rdf:resource="&fibo-fbc-fi-ip;MarketPrice"/>
+		<rdfs:label xml:lang="en">opening price</rdfs:label>
+		<skos:definition xml:lang="en">price at which a security or commodity starts a trading day</skos:definition>
+		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">Investors that want to buy or sell as soon as the market opens will put in an order at the opening price.  Depending on how the closing price for the prior day is determined, and if there is no after hours trading, the opening price will be the same as the prior trading day&apos;s closing price.</fibo-fnd-utl-av:explanatoryNote>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-fbc-fi-ip;PriceAnalytic">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-utl-alx;ScopedMeasure"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;appliesTo"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fbc-fi-ip;CollectionOfPrices"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-utl-alx;hasApplicableDatePeriod"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fnd-dt-fd;DatePeriod"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label xml:lang="en">price analytic</rdfs:label>
+		<skos:definition xml:lang="en">statistical measure involving security prices</skos:definition>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-fbc-fi-ip;PriceSpread">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-utl-alx;Difference"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-utl-alx;hasArgument"/>
+				<owl:onClass rdf:resource="&fibo-fnd-acc-cur;MonetaryPrice"/>
+				<owl:qualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">2</owl:qualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label>price spread</rdfs:label>
+		<skos:definition>difference between two prices</skos:definition>
```

Comment:
Probably, but this is simply a price spread.  One would create an individual or set of individuals of this class that constrain what that spread is about.

Reply:
Not convinced that it should be so unconstrained

Reply:
I'm with @ElisaKendall on this one; it is difficult for us to know what constraints on something like this will be meaningful.  @rivettp 's example seems evidently ridiculous, but it is hard to say that any particular pair of prices is certainly wrong (what if Apple closed its purchase of IBM at 12:30?  then these two prices might indeed be some sort of 'spread').  It seems to me that the right way to model this is what Elisa has done; model an uncontrained spread, then as subclasses of this with extra restrictions, model various known sorts of meaningful spreads.